provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerController.java

```diff
@@ -148,13 +135,105 @@ private PractitionerWrapper search(MultiValueMap<String, String> parameters) {
   public Practitioner.Bundle searchByName(
       @RequestParam("name") String name,
       @RequestParam(value = "page", defaultValue = "1") @Min(1) int page,
-      @RequestParam(value = "_count", defaultValue = "1") @Min(0) int count) {
+      @CountParameter @Min(0) int count) {
     return bundle(
         Parameters.builder().add("name", name).add("page", page).add("_count", count).build(),
         page,
         count);
   }
 
+  /** Logic for search by Family and Given. */
+  private Pair<List<PractitionerWrapper>, Integer> searchFamilyGiven(
+      MultiValueMap<String, String> parameters) {
+    String familyName = parameters.getFirst("family");
+    String givenName = parameters.getFirst("given");
+    ProviderResponse providerResponse = ppmsClient.providersForName(familyName);
+    int page = Integer.parseInt(parameters.getOrDefault("page", singletonList("1")).get(0));
```

Comment:
There was a change made in `data-query` recently to add helper methods `pageOf()` and `countOf()` to the `Parameters` class to limit proliferation of this `parseInt(...)` ugliness. We can do the same thing here. Take a look at https://github.com/department-of-veterans-affairs/health-apis-data-query/blob/master/data-query/src/main/java/gov/va/api/health/dataquery/service/controller/Parameters.java#L42